packages/suite/src/middlewares/suite/toastMiddleware.ts

```diff
@@ -36,6 +35,8 @@ const toastMiddleware =
             toast(hocNotification(payload, ToastNotification), {
                 toastId: payload.id,
                 onClose: () => api.dispatch(close(payload.id)),
+                // if 'autoclose' is not set, close notifications after 5s
+                autoClose: payload.autoClose === undefined ? 5000 : payload.autoClose,
```

Comment:
nit: it's the same as:
```suggestion
                autoClose: payload.autoClose ?? 5000,
```

Reply:
(almost the same, it will use default 5000 even if payload.autoClose === null)...

Reply:
fe338ce18c724b4e91abfe1188816e79193f903f